Creatives that Convert gated guide (#436)
readthedocs/ethicalads.io: content/pages/creatives-that-convert.md

title: Creatives that Convert
slug: creatives-that-convert
url: advertisers/creatives-that-convert/
save_as: advertisers/creatives-that-convert/index.html

> description: Whether you need help adapting ads you're running elsewhere or need new ads for your new landing pages, we can help you craft a campaign that converts.

Reviewer: ```suggestion
description: Whether you need help adapting ads you're running elsewhere or need new ads for your landing pages, we can help you craft a campaign that converts.
```

subtitle: We can help you craft a campaign that converts
template: ea/creatives-that-convert
contact_footer_subject: Get our creatives guide
contact_footer_subheadline: The guide includes even more tips and tricks to help ensure your campaign with us is successful.
contact_submit_text: Get the guide

> contact_submit_helptext: Our sales team will be in touch and send you the guide, but we take privacy seriously and don't share your info further than that.

Reviewer: ```suggestion
contact_submit_helptext: Our sales team will be in touch and send you the guide, but we take privacy seriously and won't share your information.
```
